cub/cub/block/block_merge_sort.cuh

```diff
@@ -427,12 +427,12 @@ public:
 
       int thread_idx_in_thread_group_being_merged = mask & linear_tid;
 
-      int diag = (::cuda::std::min)(valid_items, ITEMS_PER_THREAD * thread_idx_in_thread_group_being_merged);
+      int diag = (::cuda::std::min) (valid_items, ITEMS_PER_THREAD * thread_idx_in_thread_group_being_merged);
```

Comment:
I find space insertion by the linter between `(::cuda::std::min)` and its arguments questionable. 

We do not usually write `foo (x)`, but `foo(x)`.

Reply:
True but its too common and unobstrusive to stop the update